tasks/javascript.js

```diff
@@ -113,6 +115,17 @@ gulp.task('javascript:lint', function () {
 
 // karma unit tests in local library only
 gulp.task('javascript:unit', function () {
+  var reporters = cliArgs.karmareporter;
+  if (reporters.constructor === Array) {
+    reporters = Array;
+  }
+  else if (typeof reporters === 'string' &&
+      reporters !== karma.yargsOption.value.default) {
+    reporters = [reporters];
+  }
+  else {
+    reporters = [];
```

Comment:
An empty array here will result in the default reporter being run. 
This will be the case until a default karma reporter is extracted, and thus can be imported by karam as a karma plugin.
See `karmaRun`.